webpack/scenes/ContentViews/Details/Repositories/LastSync.js

```diff
@@ -1,12 +1,13 @@
 import React, { Fragment } from 'react';
 import { CheckCircleIcon, ExclamationTriangleIcon, CloseIcon } from '@patternfly/react-icons';
 import { foremanUrl } from 'foremanReact/common/helpers';
+import { translate as __ } from 'foremanReact/common/I18n';
 import PropTypes from 'prop-types';
 
 const LastSync = ({ lastSyncWords, lastSync }) => {
   if (lastSync && lastSyncWords) {
     let Icon;
-    let color = 'black';
+    let color = __('black');
```

Comment:
This will end up as a CSS style, so it shouldn't be translated

Reply:
in french it could have a nice sound: the new css `noir` :D